Web/Utility/Rest.cs

```diff
@@ -233,7 +233,7 @@ public static class Rest
         /// </param>
         /// <param name="abortToken">Cancellation token for the request.</param>
         [ItemCanBeNull]
-        public static async Task<WebRequestResponse> GetRequestAsync(
+        public static async Task<WebRequestResponseWithHandler> GetRequestAsync(
```

Comment:
This method writes the data directly to file so no handler is needed. Revert the return type back. I am using this in PolyRules and this change is breaking.